screwdriver.yaml

```diff
@@ -83,7 +83,7 @@ jobs:
     steps:
       - update-to-latest: |
           VESPA_VERSION=$(curl -sSL https://repo1.maven.org/maven2/com/yahoo/vespa/parent/maven-metadata.xml | \
-            grep '<latest>\K([0-9]+\.[0-9]+\.[0-9]+)' | sed 's/<\/*latest>//g; s/ //g;')
+            grep -oP '<latest>\K([0-9]+\.[0-9]+\.[0-9]+)' | sed 's/<\/*latest>//g; s/ //g;')
```

Comment:
No need to execute the last sed , because the grep will print only the version match.

Reply:
Thanks - learned something new there - removed in #2363